app/uk/gov/hmrc/tariffclassificationfrontend/views/reassign_queue_case.scala.html

```diff
@@ -38,7 +38,7 @@ <h3 class="heading-large mt-0">Move this case back to a queue</h3>
                     queues,
                     "reassign-case",
                     ReassignCaseController.reassignCase(c.reference),
-                    CaseController.activityDetails(c.reference)
+                    request.headers.get("Referer").getOrElse(CaseController.applicationDetails(c.reference).url)
```

Comment:
this causes an issue (found by @jumpertester)

Reply:
I think we should stick with the normal pattern for this

Reply:
There should be a view test for it anyway :p

Reply:
This is called now from multiple pages.
As a quick solution we could send the user always to the same page. 

But ideally would be nice to remember the user journey.
I thought about a solution of injecting a custom header through the requests. This header needs to be set (to the cancel URL) and reset (when the user then clicks `Cancel`)

Reply:
Sounds like something that needs to be prioritised etc @gokyo